src/app/onion/learning-object-builder/components/scaffold/hierarchy/hierarchy.component.scss

```diff
@@ -0,0 +1,80 @@
+.children-list {
+    width: 500px;
+    max-width: 100%;
+    min-height: 60px;
+    display: block;
+    background: white;
+    border-radius: 1px;
+    overflow: hidden;
+  }
+  
+  .child-box {
+    padding: 20px 10px;
+    border-bottom: solid 1px#ccc;
+    color: rgba(0, 0, 0, 0.87);
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: space-between;
+    box-sizing: border-box;
+    background: white;
+  
+    .info {
+      float: left; 
+      .top-row {
+        display: flex; 
+        .child-title {
+          color: #646464;
+          text-align: left;
+          font-size: 16px; 
+        }
+      }
+      .bottom-row{
+        display: flex; 
+        .child-length {
+          color: #646464;
```

Comment:
Why is this color hard coded? Can we use one of the defined SCSS variables?

Reply:
Resolved with Add style commit